File: apps/institution-api/Models/VwLocation.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class VwLocation
    {
        public int LocationId { get; set; }
        public string LocationName { get; set; }
        public int? LocationNumber { get; set; }
        public string LocationStatus { get; set; }
        public bool? OffSiteFlag { get; set; }
        public string LocationPspstatus { get; set; }
        public int? ProviderId { get; set; }
        public DateTime? BusinessStartDate { get; set; }
        public DateTime? BusinessEndDate { get; set; }
    }
}

File: libs/common/models/vw-location.ts



export class VwLocation {
    public locationId: number = 0;
    public locationName: string = '';
    public locationNumber: number;
    public locationStatus: string = '';
    public offSiteFlag: boolean;
    public locationPspstatus: string = '';
    public providerId: number;
    public businessStartDate: Date;
    public businessEndDate: Date;

    constructor(init? : Partial<VwLocation>) {
        
        Object.assign(this, init);
    }
}

